README.md

```diff
@@ -1,110 +1,97 @@
-# FLAMEGPU2 Ensemble Experiments
+# FLAMEGPU2 Ensemble Benchmark
 
-This repository contains the figures relating to ensembles shown in the talk *Simulate the World and Everything in It: Complex Systems Simulations with FGPU2* given by Paul Richmond at [GTC 2021](https://www.nvidia.com/en-us/gtc/topics/developer-tools/), along with the code to generate them. The code demonstrates the effect and scaling of the concurrency feature of the FLAMEGPU2 agent-based modelling framework. Please note that generated figures may differ slightly as a result of the stochastic nature of the simulations and hardware differences.
+This repository contains performance benchmarking of a [FLAME GPU 2](https://github.com/FLAMEGPU/FLAMEGPU2) implementation of Boids agent based model running multiple concurrent simulations.
 
-## Figures
+The code demonstrates the effect and scaling of the concurrency feature of the FLAMEGPU2 agent-based modelling framework.
```

Comment:
This is the ensemble benchmark not the concurrency benchmark.

Reply:
clarified